### ISSUE TITLE
These bulma components are incredibly slow with frequent rendering

### ISSUE THREAD

**Issue by mjangir** [mpaupulaire4/bulma-styled-components#11]
If I use bulma-styled-components with a controlled component where I hold the keypress in an input, then they cause a huge lagging. Instead, if I use simple divs with classes, that's too fast.

**Comment by mpaupulaire4** [comment]
Yeah this may be due to the underlying styled-component implementation. I am currently working on a newer version without the dependency (but is still compatible) that will offer performance increases

**Comment by mjangir** [comment]
@mpaupulaire4 you did a fantastic job in this project but unfortunately, I had to remove this package because its really slow with formik form component. Will surely look at in future and I hope it will have great performance.
